docs/features/software-templates/ui-options-examples.md

```diff
@@ -468,3 +468,61 @@ repoUrl:
 The supported `additionalScopes` values are `gerrit`, `github`, `gitlab`, `bitbucket`, and `azure`.
 
 If you're also using the `RepoUrlPicker` field extension, you should simply duplicate this part from there.
+
+## RepoOwnerPicker
+
+The input props that can be specified under `ui:options` for the `RepoOwnerPicker` field extension.
+
+### `host`
+
+The SCM integration host that owners should be fetched from for autocompletion.
+
+- Fetch owners from `github.com`
+
+```yaml
+repoUrl:
+  title: Repository Owner
+  type: string
+  ui:field: RepoOwnerPicker
+  ui:options:
+    host: github.com
+```
+
+This value will be overriden if a different host is specified in a `RepoUrlPicker` field extension.
```

Comment:
```suggestion
This value will be overridden if a different host is specified in a `RepoOwnerPicker` field extension.
```

Reply:
Ah sorry I misunderstood the behavior, see the most recent comment: https://github.com/backstage/backstage/pull/32105#discussion_r2622081873